doc/api/test.md

```diff
@@ -504,7 +504,7 @@ This allows developers to write more reliable and
 predictable tests for time-dependent functionality.
 
 The example below shows how to mock `setTimeout`.
-Using `.enable(['setTimeout']);`
+Using `.enable({ apis: ['setTimeout'] });`
```

Comment:
It seems like this is a `semver-major` change, isn't it?

Reply:
If the API was stable it was but this is an experimental API so I don't think it's required but I may be rememberng incorrectly

Reply:
Given this comment, I'm going to remove the https://github.com/nodejs/node/labels/semver-major label, if that's wrong, please add it back.